fastlane/lib/fastlane/plugins/template/%gem_name%.gemspec.erb

```diff
@@ -1,6 +1,4 @@
-# coding: utf-8
-
-lib = File.expand_path("../lib", __FILE__)
+lib = File.expand_path("lib", __dir__)
```

Comment:
Changes in this file are pointed by rubocop. Unlike others, this is needed to be fixed as issues will be passed to plugins newly generated😇

Reply:
This will need an update of all plugins?

Reply:
No. This is just style fix. If plugin authors think they want to follow it, they can just do it by themselves.